components/gitpod-db/src/tables.ts

```diff
@@ -204,11 +204,17 @@ export class GitpodTableDescriptionProvider implements TableDescriptionProvider
             deletionColumn: "deleted",
             timeColumn: "_lastModified",
         },
+        {
+            name: "d_b_code_sync_collection",
+            primaryKeys: ["userId", "collection"],
```

Comment:
If we added a UUID for an entry in the collection, you can then reference it as a single key in the `d_b_code_sync_resource` table rather than duplicating the `userId` and `collection` properties

Reply:
`userId` was already in `d_b_code_sync_resource` so it would be the same to use this new uuid column or the collection column :thinking:

Reply:
@jeanp413  Where is the name of `collection` stored? No need?

Reply:
profile metadata is stored in `profile` server resource, not in the collection